blog/dec-quiet-month-21.md

```diff
@@ -0,0 +1,36 @@
+---
+title: A Quiet Place (Dec'21)
+date: 2021-09-27T00:00:00.000Z
+authors:
+  name: groundwater
+  url: 'https://github.com/groundwater'
+  image_url: 'https://github.com/groundwater.png?size=96'
+slug: a-quiet-place-21
+---
+ElectronJS will pause for the month of December 2021, but will return to full speed in January 2022.
```

Comment:
We typically use `Electron` (no `js` suffix) in official communications, I think.